### ISSUE TITLE
Shared Text Styles can be updated, but no synchronisation happens in the files

### ISSUE THREAD

**Issue by sylvainaerni** [kactus-io/kactus#63]
### Description

Shared Text Styles files can be updated, but no synchronisation happens across the Sketch files 

### Version 

**Kactus version:** 0.2.10
**OS version:** OS SIerra 10.12.5

### Steps to Reproduce

1. Set the Kactus settings for Sharing Text and Layer Styles
1. Get a file with Text Styles
1. Create a new file from Kactus: the Styles are inherited
1. Modify the Text Styles into a Sketch File
1. Open another file: the Styles modifications aren't reflected

**Expected behavior:** 
When I change a Style in a Sketch file, I expect that the other files inherit from these modifications
**Actual behavior:** 
No Styles synchronisation

**Reproduces how often:** 100%

**Comment by mathieudutour** [comment]
yes, for now you need to regenerate the other sketch files. There are a lot of edge cases when re-generating the files automatically, I need to find the right spot, which I haven't yet
